docs/pages/x/api/data-grid/grid-col-def.md

```diff
@@ -14,42 +14,45 @@ import { GridColDef } from '@mui/x-data-grid';
-| <span class="prop-name optional">valueSetter<sup><abbr title="optional">?</abbr></sup></span>             | <span class="prop-type">(params: GridValueSetterParams&lt;R, V&gt;) =&gt; R</span>                                                                                                                 |                                            | Function that allows to customize how the entered value is stored in the row.<br />It only works with cell/row editing.                                            |
-| <span class="prop-name optional">width<sup><abbr title="optional">?</abbr></sup></span>                   | <span class="prop-type">number</span>                                                                                                                                                              | <span class="prop-default">100</span>      | Set the width of the column.                                                                                                                                       |
+| Name                                                                                                                                                                                                               | Type                                                                                                                                                                                               | Default                                     | Description                                                                                                                                                        |
+| :----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | :------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | :------------------------------------------ | :----------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| <span class="prop-name optional">aggregable<sup><abbr title="optional">?</abbr></sup> [<span class="plan-premium" title="Premium plan"></span>](/x/introduction/licensing/#premium-plan)</span>                    | <span class="prop-type">boolean</span>                                                                                                                                                             | <span class="prop-default">true</span>      | If `true`, the cells of the column can be aggregated based.                                                                                                        |
```

Comment:
Premium properties weren't documented here before, so this is a useful side-effect :_